packages/mint-components/src/components/sqm-coupon-code/sqm-coupon-code.feature

```diff
@@ -0,0 +1,120 @@
+@author:truman
+@owner:truman
+Feature: Coupon Code
+
+    The coupon code component is a box that allows users to see and copy their coupon code for a given program
+
+    @motivating
+    Scenario: A Users sharelink can be copied to their clipboard
```

Comment:
Why is a sharelink spec in the coupon code file? Feels like a bad copy and paste